### PR TITLE
dev-util/clion: version bump to 2018.1

### PR DESCRIPTION
**Pull request by strahlc**
- dev-util/clion: version bump to 2018.1
- dev-util/clion: remove old versions

### CODE REVIEW

File: dev-util/clion/clion-2018.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+src_install() {
+	local dir="/opt/${P}"
+
+	insinto "${dir}"
+	doins -r *
+	fperms 755 "${dir}"/bin/{clion.sh,fsnotifier{,64}}
+
+	make_wrapper "${PN}" "${dir}/bin/${PN}.sh"
+	newicon "bin/${PN}.svg" "${PN}.svg"
```
Comment (mgorny) [inline dev-util/clion/clion-2018.1.ebuild]: `doicon` will do the same.

File: dev-util/clion/clion-2018.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	use x86 || remove_me+=( plugins/tfsIntegration/lib/native/linux/x86 )
+
+	rm -rv "${remove_me[@]}" || die
+}
+
+src_install() {
+	local dir="/opt/${P}"
+
+	insinto "${dir}"
+	doins -r *
```
Comment (mgorny) [inline dev-util/clion/clion-2018.1.ebuild]: doins -r .

?

File: dev-util/clion/clion-2018.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	fperms 755 "${dir}"/bin/{clion.sh,fsnotifier{,64}}
```
Comment (mgorny) [inline dev-util/clion/clion-2018.1.ebuild]: How about `+x`?

File: dev-util/clion/clion-2018.1.ebuild (hunk 1 above)
Comment (strahlc) [reply dev-util/clion/clion-2018.1.ebuild]: done

File: dev-util/clion/clion-2018.1.ebuild (hunk 2 above)
Comment (strahlc) [reply dev-util/clion/clion-2018.1.ebuild]: done

File: dev-util/clion/clion-2018.1.ebuild (hunk 3 above)
Comment (strahlc) [reply dev-util/clion/clion-2018.1.ebuild]: done

**Comment by mgorny** [comment]
Thanks! I don't have enough free space to test it, so I'm going to trust that you've tested it yourself ;-).

**Comment by strahlc** [comment]
@mgorny yes, I did :smile:

**Pull request closed without merging**